lib/publiccloud/gce.pm

```diff
@@ -124,6 +124,7 @@ sub get_gcp_guest_os_features {
     my $os_version;
     if ($file =~ /SLES\d+-SP\d+|SLES-\d+\.\d+/i) {
```

Comment:
Only a concern about the check in the file name for the subversion: 
actual fiel name format is `sles-16-0-...`, so in line 125 the dot `.` I'm afraid should be changed to dash `-` instead.

Reply:
Dashes don't need escaping except when used inside square brackets for ranges, and this can also be avoided by placing it right before the closing bracket.